src/Microsoft.OData.Edm/Schema/EdmEntitySet.cs

```diff
@@ -41,7 +41,7 @@ public EdmEntitySet(IEdmEntityContainer container, string name, IEdmEntityType e
 
             this.container = container;
             this.type = new EdmCollectionType(new EdmEntityTypeReference(elementType, false));
-            this.path = new EdmPathExpression(this.container.FullName() + "." + this.Name);
+            this.path = new EdmPathExpression(this.container.FullName() + "/" + name);

```

Comment:
>this.container.FullName() + "/" + name [](start = 46, length = 38)

use string.Concat(this.container.FullName(), "/", name)

Reply:
change to use the name directly, so not applicable

---
In reply to: [542981676](https://github.com/OData/odata.net/pull/1950#discussion_r542981676) [](ancestors = 542981676)